### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: javascript/javascript1/week3/script1.js [hunk 1]
```diff
@@ -23,3 +23,22 @@ names.splice(indexOfName, 1);
...
+    const time = travelInformation.destinationDistance / travelInformation.speed;
+    console.log(time);
+    let hours = Math.floor(time);
+    console.log(hours);
+    let minutes = Math.floor((time - hours) * 60);
+    return hours + " Hours " + minutes + " Minutes";
+
+}
+
+const travelTime = notThisFunctionName(travelInformation);
```
Comment (andrii1) [inline javascript/javascript1/week3/script1.js:41]: I would change the name of the function :) notThisFunctionName

File: javascript/javascript1/week3/script1.js [hunk 2]
```diff
@@ -36,9 +36,78 @@ function notThisFunctionName(travelInformation) {
...
+};
+seriesDurations.push(myShow);
+console.log(seriesDurations);
+// write function to calculate how much time you spent
+
+const totalTimeInMinutes0 = (seriesDurations[0].days * 24 * 60) + (seriesDurations[0].hours * 60) + seriesDurations[0].minutes;
+const totalTimeInMinutes1 = (seriesDurations[1].days * 24 * 60) + (seriesDurations[1].hours * 60) + seriesDurations[1].minutes;
+const totalTimeInMinutes2 = (seriesDurations[2].days * 24 * 60) + (seriesDurations[2].hours * 60) + seriesDurations[2].minutes;
+const totalTimeInMinutes3 = (seriesDurations[3].days * 24 * 60) + (seriesDurations[3].hours * 60) + seriesDurations[3].minutes;
+const totalLifeSpanInMinutes = 80 * 365 * 24 * 60;
```
Comment (andrii1) [inline javascript/javascript1/week3/script1.js]: i think you can use a for loop

File: javascript/javascript1/week3/script1.js [hunk 3]
```diff
@@ -36,9 +36,78 @@ function notThisFunctionName(travelInformation) {
...
+    let totalLifeSpent3 = ((totalTimeInMinutes3 * 100) / totalLifeSpanInMinutes).toFixed(3);
+
+    if (title == seriesDurations[0].title) {
+        console.log(seriesDurations[0].title + " took " + totalLifeSpent0 + "% of my life");
+    } else if (title == seriesDurations[1].title) {
+        console.log(seriesDurations[1].title + " took " + totalLifeSpent1 + "% of my life");
+    } else if (title == seriesDurations[2].title) {
+        console.log(seriesDurations[2].title + " took " + totalLifeSpent2 + "% of my life");
+    } else if (title == seriesDurations[3].title) {
+        console.log(seriesDurations[3].title + " took " + totalLifeSpent3 + "% of my life");
```
Comment (andrii1) [inline javascript/javascript1/week3/script1.js]: I think you can combine all of this in a for loop. I did something like
`let sumPercentage = 0;
	for (let i = 0; i < seriesDurations.length; i++) {
		const timeInMinutes =
			seriesDurations[i].days * 24 * 60 +
			seriesDurations[i].hours * 60 +
			seriesDurations[i].minutes * 60;
		const percentage = (timeInMinutes / (80 * 525600)) * 100;
		console.log(
			`${seriesDurations[i].title} took ${percentage.toFixed(4)} of my life`
		);
		sumPercentage += percentage;
	}`

File: javascript/javascript1/week3/script2.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        }
+    }
+}
+let firstNote = getNote(1);
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+//Log out notes
+function logOutNotesFormatted() {
+    for (let i = 0; i < notes.length; i++) {
+        noteInFormat = console.log("The note with id: " + notes[i].id + " has the following note text: " + notes[i].content);
```
Comment (andrii1) [inline javascript/javascript1/week3/script2.js:37]: You can use something like `the note with id ${notes[i].id}`

File: javascript/javascript1/week3/script1.js (hunk 1 above)
Comment (27ManishaPatel) [reply javascript/javascript1/week3/script1.js:41]: Yes I would like to follow the commands(HYF- hw commands)

File: javascript/javascript1/week3/script1.js (hunk 3 above)
Comment (27ManishaPatel) [reply javascript/javascript1/week3/script1.js]: Yes i tried this one but still have some doubts. may be we can discuss that in class.

File: javascript/javascript1/week3/script2.js (hunk 4 above)
Comment (27ManishaPatel) [reply javascript/javascript1/week3/script2.js:37]: Yes right i will do that

File: javascript/javascript1/week3/script1.js (hunk 3 above)
Comment (shravanipagilla) [reply javascript/javascript1/week3/script1.js]: I would agree with Andrii.

File: javascript/javascript1/week3/script2.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+showStatus(activities);
+//Create a function for calculating the activity a user has spent the most time on.
+function spentMostTime(activities) {
+    let mostSpentTime = 0;
+    for (let i = 0; i < activities.length; i++) {
+        if (activities[i].Duration > mostSpentTime) {
+            mostSpentTime = activities[i].Duration;
+            mostSpentActivity = activities[i].Activity;
+        }
+    }
```
Comment (shravanipagilla) [inline javascript/javascript1/week3/script2.js:100]: it is very great work manisha,  you put lots of efforts. overall well done manisha, just go with small changes .

File: javascript/javascript1/week3/script1.js (hunk 2 above)
Comment (27ManishaPatel) [reply javascript/javascript1/week3/script1.js]: yes I did it and it works good thanks

File: javascript/javascript1/week3/script1.js (hunk 3 above)
Comment (27ManishaPatel) [reply javascript/javascript1/week3/script1.js]: Yes i made a loop and its works now thanks both of you

**Pull request merged**